charts/seed-controlplane/charts/cloud-controller-manager/templates/alicloud-ccm-deployment.yaml

```diff
@@ -0,0 +1,107 @@
+{{- if eq .Values.cloudProvider "alicloud" -}}
```

Comment:
As already discussed in person: Please do not copy the whole resource manifest of CCM just for alicloud and separate with `if`/`else`. To me it looks that the majority of the specification is the same compared to all the other cloud providers.
Usually, the resource manifests should not contain `if`/`else` constraints regarding cloud differences - just regarding version differences ([example](https://github.com/gardener/gardener/blob/master/charts/seed-controlplane/charts/cloud-controller-manager/templates/cloud-controller-manager.yaml#L74-L79)). Please let the alicloud botanist specify the flags (that's should be the only cloud specific implementation), similar to https://github.com/gardener/gardener/blob/master/pkg/operation/cloudbotanist/awsbotanist/controlplane.go#L131-L136. The CCM Helm chart takes e.g. https://github.com/gardener/gardener/blob/master/charts/seed-controlplane/charts/cloud-controller-manager/templates/cloud-controller-manager.yaml#L85-L87 that you can use.

Reply:
got it. will follow up the rule.

Reply:
I did another investigation, and found there is quite difference in Starting command args for the 2 CCM. I think it is a better way to seperate 2 CCM. Please check my latest commit.